src/cmd/cmd.cpp

```diff
@@ -467,14 +443,13 @@ int main(int argc, char **argv) {
 
     Cmd cmd;
     SyncJournalDb db(options.source_dir);
-    selectiveSyncFixup(&db, selectiveSyncList);
+    if (!selectiveSyncList.empty())
+        selectiveSyncFixup(&db, selectiveSyncList);
```

Comment:
Yes. This change the behaviour and it is intentional.  
I think the behaviour is better like that where the option allow to change the blacklist rather than using it.